main.md

```diff
@@ -929,6 +929,17 @@ the authorization server MUST reject the request.
 The rejection response MAY include a `DPoP-Nonce` HTTP header
 providing a new nonce value to use for subsequent requests.
 
+The intent is that both clients and servers need to keep only one nonce value for one another.
+That said, transient circumstances may arise in which the server's and client's
+stored nonce values differ.
+The good news is that this situation is self-correcting.
```

Comment:
The phrase "The good news is" is a bit folksy for a spec. 
Maybe combine the two sentences here to something like:
"However, this situation is largely self-correcting because, with any rejection message, the server can send the client the ..."

Reply:
Changed as suggested.